test/src/main/java/org/apache/accumulo/test/ClientSideIteratorIT.java

```diff
@@ -125,20 +125,25 @@ public void testVersioning() throws Exception {
     bw.addMutation(m);
     bw.flush();
 
-    final Scanner scanner = conn.createScanner(tableName, new Authorizations());
-
-    final ClientSideIteratorScanner csis = new ClientSideIteratorScanner(scanner);
-    final IteratorSetting si = new IteratorSetting(10, "localvers", VersioningIterator.class);
-    si.addOption("maxVersions", "2");
-    csis.addScanIterator(si);
-
-    checkResults(csis, resultSet1, PartialKey.ROW_COLFAM_COLQUAL_COLVIS_TIME);
-    checkResults(scanner, resultSet2, PartialKey.ROW_COLFAM_COLQUAL_COLVIS_TIME);
-
-    csis.fetchColumnFamily(new Text("colf"));
-    checkResults(csis, resultSet1, PartialKey.ROW_COLFAM_COLQUAL_COLVIS_TIME);
-    csis.clearColumns();
-    csis.fetchColumnFamily(new Text("none"));
-    assertFalse(csis.iterator().hasNext());
+    ClientSideIteratorScanner csis = null;
+    try (Scanner scanner = conn.createScanner(tableName, new Authorizations())) {
```

Comment:
It's possible to use try-with-resources for both `scanner` and `csis` in the same block. I don't recall if the separating character is a semicolon or a comma.

```java
try (Scanner scanner = conn.createScanner(tableName, Authorizations.EMPTY); ClientSideIteratorScanner csis = new ClientSideIteratorScanner(scanner)) {
   // ...
}
```

You can also nest them, but that's uglier.